modules/rounds.js

```diff
@@ -387,6 +387,16 @@ __private.getOutsiders = function(scope, cb, tx) {
  * @todo Add description for the params and the return value
  */
 __private.sumRound = function(scope, cb, tx) {
```

Comment:
The `__private.sumRound` is triggered from `tick`. There is a condition. 

```
// Establish if finishing round or not
	scope.finishRound =
		round !== nextRound || (block.height === 1 || block.height === 101);
```

I can't understand why this condition contains `|| block.height === 101`

Reply:
Out of scope of this PR, I didn't touch that part.

Reply:
But you touch the logic of `genesisBlock` to be round 1. And I believe this code relates to the logic.

Reply:
No, I didn't changed the logic - round 1 was allways summed twice:
- just after genesis block (to elect delegates, update their vote weight, etc.)
- after 101 blocks (actual round 1 ended)
To be honest that condition probably do nothing, but I don't really want to touch that part there, as is out of scope of the issue.